doc/releases/release-notes-3.6.rst

```diff
@@ -346,6 +346,10 @@ Libraries / Subsystems
 
   * File systems: LittleFS module has been updated to version 2.8.1.
 
+  * Following Flash Map API macros, marked in 3.2 as deprecated, have been removed:
+    :c:macro:`FLASH_AREA_ID`, :c:macro:`FLASH_AREA_OFFSET`, :c:macro:`FLASH_AREA_SIZE`,
+    :c:macro:`FLASH_AREA_LABEL_EXISTS` and :c:macro:`FLASH_AREA_DEVICE`.
```

Comment:
You need to use double backticks instead as these macros are removed.

Reply:
Thanks, fixed.